tests/test_klaviyo_start_date.py

```diff
@@ -15,18 +15,33 @@ def name():
         return "tap_tester_klaviyo_start_date_test"
 
     def test_run(self):
+
+        # We are not able to generate test data so skipping two streams(mark_as_spam, dropped_email)
+        # Skipping given streams due to data available only on a single date -  failed_to_deliver_automated_response, failed_to_deliver
+        
+        expected_streams = self.expected_streams() - {"mark_as_spam", "dropped_email", "failed_to_deliver_automated_response", 
```

Comment:
Since streams are being omitted here, can you remove the skip condition on line 96

Reply:
Removed the skip condition.